### ISSUE TITLE
Pepper flash crash

### ISSUE THREAD

**Issue by Inixi** [gcarq/inox-patchset#57]
Hello,
After last update browser always crash after loading a page with flash player. After entering page with flash player (e.g. http://www.adobe.com/software/flash/about/) following stack trace generates:
```
A Parser-blocking, cross-origin script, https://static.adobelogin.com/imslib/imslib.min.js, is invoked via document.write. This may be blocked by the browser if the device has poor network connectivity. See https://www.chromestatus.com/feature/5718547946799104 for more details.
A Parser-blocking, cross-origin script, https://static.adobelogin.com/imslib/imslib.min.js, is invoked via document.write. This may be blocked by the browser if the device has poor network connectivity. See https://www.chromestatus.com/feature/5718547946799104 for more details.
[WARNING:flash/platform/pepper/pep_module.cpp(63)] SANDBOXED
Vector smash protection is enabled.
Received signal 11 SEGV_MAPERR 000000000018
#0 0x55eeaddc972e <unknown>
#1 0x55eeaddc9ae9 <unknown>
#2 0x7f7d1ea70080 <unknown>
#3 0x55eeb11dc4f1 <unknown>
#4 0x55eeade51844 <unknown>
#5 0x55eeafad1161 <unknown>
#6 0x55eeafad1ad4 <unknown>
#7 0x55eeade51844 <unknown>
#8 0x55eeaddea9d0 <unknown>
#9 0x55eeaddec65d <unknown>
#10 0x55eeaddecb00 <unknown>
#11 0x55eeadded969 <unknown>
#12 0x55eeadde9a75 <unknown>
#13 0x55eeade0b50a <unknown>
#14 0x55eeb1113cb2 <unknown>
#15 0x55eeada825f1 <unknown>
#16 0x55eeada82af8 <unknown>
#17 0x55eeada80e09 <unknown>
#18 0x55eeac623f6c ChromeMain
#19 0x7f7d1456e291 __libc_start_main
#20 0x55eeac623e2a _start
  r8: 0000000000000003  r9: 0000000000000031 r10: 00007ffc20bd38a0 r11: 000055eeb3f27590
 r12: 0000000000000000 r13: 00007ffc20bd38c0 r14: 00007ffc20bd3890 r15: 000055eeb21d9513
  di: 0000014824d6bbb0  si: 0000014824b7f210  bp: 00007ffc20bd38e0  bx: 00007ffc20bd3a10
  dx: 0000000000000001  ax: 0000000000000000  cx: 0000000000000018  sp: 00007ffc20bd3730
  ip: 000055eeb11dc4f1 efl: 0000000000010206 cgf: 0000000000000033 erf: 0000000000000004
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000018
```
Is this an issue of Pepper Flash ? Probably, but on native Chromium flash player works, there is neither a stack trace nor crash.
My software:
inox 56.0.2924.76-2
pepper-flash 24.0.0.194-1

**Comment by gcarq** [comment]
Thanks for reporting. I think I messed something up during the update, flash itself should be fine.
I'll look into this when I'm at home.

**Comment by gcarq** [comment]
I think  [this upstream patch](https://github.com/gcarq/inox-patchset/blob/master/fix-building-without-webrtc.patch) causes the issue, because it's the only part where pepper code is touched. If pepper flash works with `enable_webrtc=true` it's most likely an upstream bug.

**Comment by Inixi** [comment]
I will try to compile it without the said patch and test the browser and I will give you the feedback.
P.S. it will take some time, because I am using aur's precompiled package...

**Comment by gcarq** [comment]
Oh that will probably take a long time. I will test it in a few hours with a local cache anyway, but feel free to get dirty with the build process :)

**Comment by gcarq** [comment]
The said patch caused the trouble. With `enable_webrtc=true` pepper-flash 24.0.0.194-1 runs as expected. I will report this issue upstream.

**Comment by Inixi** [comment]
I don't have resources to check it now, thanks!

**Comment by gcarq** [comment]
Should be fixed with [57.0.2987.133](https://github.com/gcarq/inox-patchset/releases/tag/57.0.2987.133).
